wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -529,6 +531,29 @@ def run(self):
                 data_for_new_model = self.get_data_for_new_model(
                     serialized_data, mapped_import_fields, record_id
                 )
+                # extract m2m fields to avoid getting the error
+                # direct assignment to the forward side of a many-to-many set is prohibited
+                m2m_fields = {}
+                for field_name, value in data_for_new_model.items():
```

Comment:
We do this loop to identify m2m fields several times in this file - could we break it out to a/several repeatable functions?